app/ResolveRoute.js

```diff
@@ -7,8 +7,9 @@ export const routeRegex = {
     CategoryFilters: /^\/(hot|created|trending|active|promoted)\/?$/ig,
     PostNoCategory: /^\/(@[\w\.\d-]+)\/([\w\d-]+)/,
     Post: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)\/?($|\?)/,
+    PostJson: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)(\.json)$/,
     UserJson: /^\/(@[\w\.jso\d-]+)(\.json)$/,
```

Comment:
It originated on your other PR, but I didn't catch it then: https://github.com/steemit/steemit.com/pull/1120/files#diff-01d5da383557e7bed8e88679f4f14e94R10